libs/ledger-live-common/src/__tests__/accounts/groupPerDay.ts

```diff
@@ -1,6 +1,9 @@
 import flatMap from "lodash/flatMap";
 import { fromAccountRaw, groupAccountOperationsByDay } from "../../account";
 import { TezosAccountRaw } from "../../families/tezos/types";
+import { setSupportedCurrencies } from "../../currencies";
+setSupportedCurrencies(["tezos"]);
+
```

Comment:
Surprised that's required here :thinking: can you explain the reason?

Reply:
For some reason the test is breaking with an error "Tezos currency not supported", I fixed this the same way for algorand with @hzheng-ledger